ios/install.md

```diff
@@ -1,8 +1,21 @@
 # Installation Process
 
 Notes:
+
 * `react-native ^v0.4.3` is required
 
+## Use CocoaPods
+
+If you already use [CocoaPods](https://cocoapods.org/) in your react-native project you add the native
+part of Mapbox GL with just one line of code:
+
+1. `npm install react-native-mapbox-gl --save`
+1. Add `pod 'RCTMapboxGL', :path => 'node_modules/react-native-mapbox-gl/ios'` to your `Podspec` file and re-run `pod install`.
```

Comment:
By the way, this points to a local, npm-based installation of RCTMapboxGL. I guess it isn’t strictly necessary to install the pod from cocoapods.org.